haskell-cabal.el

```diff
@@ -488,7 +489,10 @@ OTHER-WINDOW use `find-file-other-window'."
                           (string= val "{")
                           (not val))
                       (push projectName matches)
-                    (push val matches))))
+                    (push (if (eq 'stack-ghci process-type)
+                              (concat projectName ":" val)
```

Comment:
oh I see, then clearly `projectName` is a misnomer (but your PR didn't introduce it... it was already in the codebase before that... :-) )